springfox-swagger2/src/test/groovy/springfox/documentation/swagger2/mappers/ModelMapperSpec.groovy

```diff
@@ -94,17 +102,21 @@ class ModelMapperSpec extends SchemaSpecification {
       Map<String, Model> modelMap = modelProvider.dependencies(
         inputParam(
             "group",
-            listOfMapOfStringToString(),
+            resolver.resolve(listOfMapOfStringToString()),
+            Optional.absent(),
+            new HashSet<>(),
             DocumentationType.SWAGGER_2,
+            typeNameAdjuster,
             alternateTypeProvider(),
             namingStrategy,
             ImmutableSet.builder().build()))
     when:
       def mapped = Mappers.getMapper(ModelMapper).mapModels(modelMap)
     then:
-      mapped.containsKey("MapOfstringAndstring")
+      mapped.containsKey("1484189964")
```

Comment:
This seems like it will make things very difficult to troubleshoot. Is there a better way to do this?

Reply:
`Id` - is a ```hashCode()``` of the `ModelContext`
`name` - is a normal human readable name of the model.

We need id's to differ models if they have same name.